torch/optim/adam.py

```diff
@@ -181,7 +188,8 @@ def adam(params: List[Tensor],
          lr: float,
          weight_decay: float,
          eps: float,
-         maximize: bool):
+         maximize: bool,
+         capturable: bool):
```

Comment:
I don't think you will need this here. Only the type of the inputs will be different.

Reply:
On second thought, I prefer to keep the explicit `capturable` argument: it allows more rigorous error checking.